server/controllers/propertyController.js

```diff
@@ -161,35 +162,34 @@ class PropertyController {
 		});
 	}
 
-
 	// update property details
 
-	static updatePropertyDetails(req, res) {
-		const is_admin = req.user.is_admin;
-		const ownerID = req.user.id;
+	static async updatePropertyDetails(req, res) {
+		const { price, state, city, address, type } = req.body
+		const is_admin = req.user.rows[0].is_admin;
```

Comment:
you can use object and array destructuring to get both the admin and the id from `rows[0]`